### ISSUE TITLE
When is GA scheduled for v5 Microsoft.Azure.Functions.Worker.Extensions.ServiceBus nuget package?

### ISSUE THREAD

**Issue by willyt150** [Azure/azure-functions-dotnet-worker#911]
Hello,

Would you be able to point me to/let me know what the planned schedule is for a GA release of the v5 Microsoft.Azure.Functions.Worker.Extensions.ServiceBus nuget package is planned? It's been 7 months since beta.6 was released and we're really really liking the ability to trigger our azure functions off the service bus while using the managed identity and not a connection string.

Thanks!

**Comment by kshyju** [comment]
@willyt150 We published 5.4.0 version with the latest bits and is now available in the nuget gallery. Please give it a try and provide feedback if you have any.

https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.ServiceBus/5.4.0

These worker extension releases are aligned with the webjobs version of the package. Please check the [azure-sdk-for-net repo for the release details.](https://github.com/Azure/azure-sdk-for-net/releases?q=WebJobs.Extensions.ServiceBus&expanded=true)

**Comment by willyt150** [comment]
Thanks @kshyju will definitely take a look.

Sorry I'm still confused on the release alignment. The link you provided shows that the WebJobs version of the package had a GA release back 10/26/21 and then before that a pre-release in 3/23/21, does that mean that we should expect only GA releases of the package around October each year and a pre-release around March?

**Comment by fabiocav** [comment]
@willyt150 you're correct that today, some of those extension updates are lagging behind, but this is something we're actively working on addressing with the teams who own the various extensions.
